kong/conf_loader/init.lua

```diff
@@ -903,25 +903,28 @@ local function check_and_infer(conf, opts)
   if conf.lua_ssl_trusted_certificate then
     local new_paths = {}
 
-    for i, path in ipairs(conf.lua_ssl_trusted_certificate) do
+    for _, path in ipairs(conf.lua_ssl_trusted_certificate) do
       if path == "system" then
         local system_path, err = utils.get_system_trusted_certs_filepath()
         if system_path then
           path = system_path
 
         else
-          errors[#errors + 1] =
-            "lua_ssl_trusted_certificate: unable to locate system bundle - " ..
-            err
+
+          log.info("lua_ssl_trusted_certificate: unable to locate system bundle: " ..
```

Comment:
warn?

Reply:
No, otherwise we will get warnings in our own images, some of which don't have ssl certs.